docs/README.md

```diff
@@ -0,0 +1,70 @@
+[![CircleCI](https://circleci.com/gh/scalar-labs/scalardl-javascript-sdk-base/tree/master.svg?style=svg)](https://circleci.com/gh/scalar-labs/scalardl-javascript-sdk-base/tree/master)
+
+NPM package `@scalar-labs/scalardl-javascript-sdk-base` is the common part for package [@scalar-labs/scalardl-web-client-sdk](https://github.com/scalar-labs/scalardl-web-client-sdk) and [@scalar-labs/scalardl-node-client-sdk](https://github.com/scalar-labs/scalardl-node-client-sdk).
+Although those two packages use different gRPC tools to generate the service and the Protobuf objects, they can use @scalar-labs/scalardl-javascript-sdk-base after they inject the objects. We will introduce how to generate related static files later in this README.
+
+![relationship](./README.png)
```

Comment:
On the symbolic link to the `README.md` in the root folder, this image is broken. (The `docs/README.md` file looks OK though.) 

To ensure that the image appears in both places, maybe we should use an absolute URL here. What do you think?

Reply:
My bad.
I missed this one.
I fixed in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/91/commits/966ee8c97c3e9c8471e0ab2a0382f8146c8d8029

Reply:
My bad.
I missed this one.

I revised it in https://github.com/scalar-labs/scalardl-javascript-sdk-base/pull/91/commits/e181e53bf92ebdc1951c3d5112f73fd765bcb05c

(Thi URL is not available right now but should be OK after merged)